lib/api_constraints.rb

```diff
@@ -1,14 +1,27 @@
 class ApiConstraints
-  def initialize(options)
-    @version = options[:version]
-    @default = options[:default]
+  def initialize(version:)
+    @version = version
   end
 
   def matches?(req)
-    # API versioning isn't something that we've made a decision on yet, so this might be subject
-    # to change.
-    # Currently the only client of the API is TradeTariffWebApp and we are free to evolve both the
-    # server and the client in lockstep.
-    @default || req.headers['Accept'].include?("application/vnd.uktt.v#{@version}")
+    is_default? || req.headers['Accept'].include?("application/vnd.uktt.v#{@version}")
+  end
+
+  private
+
+  def is_default?
+    if Rails.env.production?
+      production_default?
+    else
+      development_default?
+    end
+  end
+
+  def development_default?
+    @version == 2
+  end
+
+  def production_default?
+    @version == 1
   end
 end
```

Comment:
Are we sure we want that different based on behavior in prod/dev ?
it could bring to nasty surprises

It sounds like we could set an ENV var, for example `default_api_ver `, which we set only in local,  without leverage the dev/prod

Reply:
maybe we would need only a change like this:
```
  def initialize(options)
    @version = options[:version]
    @default = options[:default]

    @default.merge(default: ENV('api_ver_default')) if ENV('api_ver_default')
  end
```